### DISCUSSION TITLE
Moving all front-end component to SiPPOB-Vue folder

### DISCUSSION THREAD

**Discussion by ZumaAkbarID** [ZumaAkbarID/SiPPOB#18] [Announcements]
### GAISSSS

Nanti buat perubahannya **ga lagi di folder Front-End**
tapi langsung ke folder SiPPOB-Vue yaa, karena udh ku satuin

@dicorzki @ZumaAkbarID @praditus343 @Syamsoelll @RachmasariAR @DennitaNF

**Comment by Syamsoelll** [comment]
mangeak
